master/getting-started/kubernetes/upgrade/downgrade.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Downgrading Calico
+no_canonical: true
+---
+
+
+Under some circumstances, you may need to perform a downgrade and return your
+cluster to the previous version of {{site.prodname}}. You may need to do this
+before running `calico-upgrade complete` or afterwards. If you need to downgrade 
+your cluster after running `calico-upgrade complete`, you should do so as soon
+as possible to avoid an outage. Any endpoints created after `calico-upgrade complete`
```

Comment:
It's not just endpoints though. 

It's really "any pods created after will not work, and any configuration or policy changes made after will not be downgraded to v2.6"